web/src/stream_popover.js

```diff
@@ -552,6 +553,20 @@ export async function build_move_topic_to_stream_popover(
         $("#move_topic_form .move_messages_edit_topic").trigger("focus");
     }
 
+    function get_count_of_messages_to_be_moved(selected_option) {
+        if (selected_option === "change_one") {
+            return 1;
+        } else if (selected_option === "change_later") {
```

Comment:
No need of `else if` here, just `if` should work since the previous block returns.